specs/paths/statements/references/list.json

```diff
@@ -28,7 +28,7 @@
         ],
         "requestBody": { "$ref": "../../../global/requests.json#/Reference" },
         "responses": {
-            "200": { "$ref": "../../../global/responses.json#/ReferenceHash" },
+            "200": { "$ref": "../../../global/responses.json#/SnakMap" },
```

Comment:
For some reason I don't actually see the Snaks in the SnakMap (We might need to expand the example)

Reply:
I guess this is down to the fact that I didn't go all the way down the tree of JSON when writing the example.
I couldn't decide at the time if it was worth it or not (as it would be very verbose).
We could touch back on this when we go through and expand all examples? (Or I can expand it now)